src/background/extension.ts

```diff
@@ -77,7 +77,9 @@ export class Extension {
     private awaiting: (() => void)[];
 
     async start() {
-        await this.config.load({local: true});
+        if (this.user.settings.syncSitesFixes) { 
```

Comment:
Preserve `await this.config.load({local: true});` at the beginning (no matter of the setting). If there will be a network error for remote config, the local one will be used and the app will not break.